src/clinical/clinical-service.ts

```diff
@@ -425,7 +426,11 @@ export const getValidRecordsPostSubmission = async (
 
 		const migrationDictionary = await dictionaryManager
 			.instance()
-			.loadSchemaByVersion(schemaName, migrationVersion);
+			.loadSchemaByVersion(schemaName, migrationVersion)
+			.catch(async (err) => {
+				L.error('getValidRecordsPostSubmission error finding migration schema', err);
+				return await dictionaryManager.instance().getCurrent();
+			});
```

Comment:
If there's an error loading a specific version of the dictionary then we default to the current?
This seems like new functionality/breaking change. cc @joneubank for 2nd pair of eyes

Reply:
This is due to an edge case I'm encountering on localhost; arguably should be a separate PR, but it could cause further Submitted Data issues (has slowed me down) which is why I included it here

When the requested migration dictionary is not found, this breaks the Clinical Data query. So I've added `getCurrent` as a fallback.
This is a problem with my local DB getting out of sync w/ Lectern Dev somehow; however it's also an edge case I'd hate to see in Prod or Staging

Reply:
Sorry to respond so late after this is merged. I am surprised a data query is reliant on fetching a dictionary from Lectern. All dictionaries that we use, should be already loaded into the clinical DB - is that not the case in this instance? This shouldn't be dependent on fetches out to lectern, just read the relevant dictionary from the DB.

Reply:
Right we did chat about this -- I will make a quick PR to revise. For now this unblocks local dev using Submitted Data.